bagitobjecttransfer/recordtransfer/templates/recordtransfer/transferform_group.html

```diff
@@ -1,45 +1,45 @@
 {% extends 'recordtransfer/transferform_base.html' %}
 {% load static %}
 {% load i18n %}
-
 {% block buttonarrayclass %}flex-item{% endblock %}
 {% block formfields %}
+    <script>
+        const ID_DISPLAY_GROUP_DESCRIPTION = "{{ ID_DISPLAY_GROUP_DESCRIPTION }}";
+        const ID_SUBMISSION_GROUP_SELECTION = "{{ ID_SUBMISSION_GROUP_SELECTION }}";
+        const fetchUsersGroupDescriptionsUrl = "{% url 'recordtransfer:get_users_group_descriptions' user_id=request.user.id %}";
+        const DEFAULT_GROUP_ID = {% if DEFAULT_GROUP_ID is not None %}"{{ DEFAULT_GROUP_ID }}"{% else %}null{% endif %};
+    </script>
```

Comment:
I suggest moving this to `{% block javascript %}` as well. Make sure it's not in `{% block formfields %}`

Also: rename `'recordtransfer:get_users_group_descriptions'` to the new name of that URL (see urls.py).

Reply:
Resolved by:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/bf1d9d5a012c5a2c091b4604b858c9350c06a4b3
and 
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/b60e49d6b068284632a665faf533ec11b1fd0a0b